### DISCUSSION TITLE
Github Education Expiration

### DISCUSSION THREAD

**Discussion by kwhandy** [community/community#30170] [GitHub Education] [answered]
Hi guys, anyone know the end date terms of benefits we've received as student(include Copilot, Github Pro, Sentry, etc that expire "as long as you're student")? I'm going through all over the internet forest but still can't find the point.

## My case
1. I've submitted the application at Oct 5, 2020
2. Got welcome notification at Nov 1, 2020
3. The expiration date at { i.dont.know }

Anyone can help me?

**Comment by radar-blip** [comment]
Dont know if this will help you at all, just happen to come across it a few minutes ago (a seperate discussion).
Read through it if you want: #30048

**Reply by Zoltus** [reply]
> Dont know if this will help you at all, just happen to come across it a few minutes ago (a seperate discussion). Read through it if you want: #30048

Thats a bug on that thread linux

[veverkap](https://github.com/veverkap)
[20 hours ago](https://github.com/orgs/community/discussions/30048#discussioncomment-3430929)
I can confirm that you both will continue having access after August 22nd. There is a bug (that we are fixing) for some users that shows this incorrect information.

I'm sorry for the inconvenience and will let you know when it is fixed.

**Reply by Nevil-90** [reply]
Have a doubt, my friend applied and got approved on April month, my friend college ID stoped on June 2025, so does the Copilot Pro still remains?

**Comment by Zoltus** [comment, marked as answer]
> Hi guys, anyone know the end date terms of benefits we've received as student(include Copilot, Github Pro, Sentry, etc that expire "as long as you're student")? I'm going through all over the internet forest but still can't find the point.
> 
> ## My case
> 1. I've submitted the application at Oct 5, 2020
> 2. Got welcome notification at Nov 1, 2020
> 3. The expiration date at { i.dont.know }
> 
> Anyone can help me?

Your Github student licence lasts for 2 years.
So yours will end at  Nov 1, 2022
You will get notification on your email soon that its ending.
And you can reapply when it ends if you are still a student!

**Reply by asmahdi08** [reply]
Will I get to claim the previously claimed limited-time offers like the 1-year domains again?

**Reply by prjailani** [reply]
Yeah. Same doubt.

**Reply by iamreiyn** [reply]
No, it varies. For example, you won't be able to renew your Namecheap domain as it is for new people who haven't used it before. I would recommend contacting the support of the benefit offering service to avoid any misconceptions or to be aware of their offer eligibility

**Reply by x7dl8p** [reply]
so what did you found out ?

**Comment by nvrzqsmv** [comment]
Hello. My student ID has been approved. And when can I enjoy the benefits?

**Reply by nibivid** [reply]
check the billing of github

**Comment by romanyn36** [comment]
I wonder what happened to the student back with you after two years ⁦:⁠-⁠)⁩

**Comment by asmahdi08** [comment]
So I had github student dev pack way back. It expired after 2 years, after that I reapplied and got it again
